src/Reflection/DocBlock/Tag/MethodTag.php

```diff
@@ -8,45 +8,30 @@
 
 class MethodTag implements TagInterface, PhpDocTypedTagInterface
 {
-    /**
-     * Return value type
-     *
-     * @var string[]
-     * @psalm-var list<string>
-     */
+    /** @var list<string> */
     protected $types = [];
 
-    /** @var string */
+    /** @var string|null */
     protected $methodName;
 
-    /** @var string */
+    /** @var string|null */
     protected $description;
 
-    /**
-     * Is static method
-     *
-     * @var bool
-     */
+    /** @var bool */
     protected $isStatic = false;
 
-    /**
-     * @return string
-     */
+    /** @return 'method' */
     public function getName()
     {
         return 'method';
     }
 
-    /**
-     * Initializer
-     *
-     * @param  string $tagDocblockLine
-     */
-    public function initialize($tagDocblockLine)
+    /** {@inheritDoc} */
```

Comment:
Since this should be importing the whole docBlock, I would make it explicit https://github.com/phpDocumentor/fig-standards/blob/master/proposed/phpdoc-tags.md#41-making-inheritance-explicit-using-the-inheritdoc-tag.

Reply:
dd68158de612598d53a7f3dcc96c27c5b3816dbf